cookbook/custom-page-data-provider.rst

```diff
@@ -7,28 +7,55 @@ comes with a few included providers and also allows to implement new ones to loa
 
 In some cases, it might be useful to register an additional provider, which only returns pages including a property
 with a specific value. This is possible by registering a new instance of the ``PageDataProvider`` included in Sulu
-that uses a custom impementation of the ``QueryBuilder``. To do this, the following changes are required:
+that uses a custom impementation of the ``QueryBuilder``. In the following example a custom ``AuthorPageDataProvider``
+is implemented, which filters the pages for a specific author:
 
 **1. Implement a custom** ``QueryBuilder`` **class that adds conditions**
 
 .. code-block:: php
-
     <?php
-    // src/SmartContent/ProductPageQueryBuilder.php
+    // App\SmartContent\AuthorPageQueryBuilder
```

Comment:
this is the filename that should be created 🙂 

```suggestion
    // src/SmartContent/AuthorPageQueryBuilder.php
```

Reply:
Yup I knew that, no idea why I copied the namespace 🙈